knimin/tests/test_ag_search.py

```diff
@@ -64,7 +65,9 @@ def test_post_barcode(self):
                              field == 'ag_kit_barcode_id'):
                             self.assertNotIn(sample[field], response.body)
                         else:
-                            self.assertIn(sample[field], response.body)
+                            exp = xhtml_escape_recursive(sample[field])
```

Comment:
If @josenavas is fine with this, then OK, cause IMOO this defeats the idea of deterministic tests and basically if some code changes something in the DB, the test will not catch it; but perhaps this is fine for this projects ...